packages/svelte/tests/runtime-runes/samples/invalid-html-ssr/_config.js

```diff
@@ -0,0 +1,36 @@
+import { test } from '../../test';
+
+let console_error = console.error;
+
+/**
+ * @type {any[]}
+ */
+const log = [];
+
+export default test({
+	compileOptions: {
+		dev: true
+	},
+
+	html: `<p></p><h1>foo</h1><p></p>`,
+
+	recover: true,
+
+	before_test() {
+		console.error = (x) => {
+			log.push(x);
+		};
+	},
+
+	after_test() {
+		console.error = console_error;
+		log.length = 0;
+	},
+
+	async test({ assert, target }) {
+		assert.htmlEqual(target.innerHTML, `<p></p><h1>foo</h1><p></p>`);
+		if (log.length > 0) {
+			assert.deepEqual(log, ['Svelte SSR validation error:\n\n']);
+		}
```

Comment:
The error should show up when mode is `hydrate`, testing if the log is present could result in false positive tests passing (in fact: the `deepEqual` should never succeed because it's not the full error message)